hydromt/models/model.py

```diff
@@ -157,17 +186,19 @@ def __getattr__(self, name: str) -> ModelComponent:
         return self._components[name]
 
     @property
-    def region(self) -> ModelRegionComponent:
-        """Return the model region component."""
-        return self.get_component("region", ModelRegionComponent)
+    def region(self) -> SpatialModelComponent:
```

Comment:
I wonder if here we don't want to return a geodataframe similar to the spatialcomponent.region property? I dont think we need the other properties of the component now anymore (eg _data etc, also as crs can be derived from the geodataframe which maybe our model should still have? a crs property I mean)